.travis.yml

```diff
@@ -16,6 +18,5 @@ before_cache:
 before_install:
   - bash setup-travis.sh
 script:
-  - sbt test
-  - sbt cli/nativeLink
-  - LD_LIBRARY_PATH=libhttpsimple/target cli/target/scala-2.11/rp --help
+  - sbt clean test package buildAll
```

Comment:
Travis builds for all distros. If this is too slow in practice, maybe we'll just build one rpm and one deb.